src/layout/themes.ts

```diff
@@ -34,6 +34,9 @@ export const darkTheme: RaThemeOptions = {
         },
       },
     },
+    MuiFilledInput: {
```

Comment:
The custom `darkTheme` here extends react-admin's `defaultTheme`, which is in fact the `defaultLightTheme` and contains light-theme specific changes for inputs that should not apply to dark theme.